virtual-server/examples/python/kubevirtclient.py

```diff
@@ -0,0 +1,269 @@
+import os
+import six
+
+from kubernetes import client, config, watch
+from kubernetes.client.exceptions import (
+    ApiTypeError,
+    ApiValueError
+)
+
+
+class KubeVirtClient:
+    """
+    Since native python client for kubevirt (https://github.com/kubevirt/client-python) has unresolved issues,
```

Comment:
Just changing the language a bit:
"Since the native python client for kubevirt (https://github.com/kubevirt/client-python) has yet unresolved issues, this class demonstrates basic operations utilizing the kubevirt api."